src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArr = [];
+  let newState = { ...state };
```

Comment:
Let's rename these variables

```suggestion
  const stateHistory = [];
  let stateCopy = { ...state };
```